tianshou/__init__.py

```diff
@@ -1,7 +1,7 @@
 from tianshou import data, env, utils, policy, trainer, exploration
 
 
-__version__ = "0.2.7"
+__version__ = "0.3.0a0"
```

Comment:
`rc` instead of `a` is more usual in my opinion.

Reply:
```suggestion
__version__ = "0.3.0rc0"
```